lib/ex_unit/lib/ex_unit.ex

```diff
@@ -177,7 +177,7 @@ defmodule ExUnit do
   If you want to run tests manually, you can set the `:autorun` option
   to `false` and use `run/0` to run tests.
   """
-  @spec start(Keyword.t()) :: nil | list(fun)
+  @spec start(Keyword.t()) :: nil | :ok
```

Comment:
Do we need to know if this returns `nil` or `:ok`, or can we just force the return of `:ok`? If we need the two return values I think we should document what they mean.

Reply:
We can definitely force the return of `:ok` here.